python/tvm/driver/tvmc/compiler.py

```diff
@@ -95,6 +95,12 @@ def add_compile_parser(subparsers):
         type=common.parse_shape_string,
         default=None,
     )
+    parser.add_argument(
+        "--disabled-pass",
+        help="disable specific passes, comma-separated list of pass names",
```

Comment:
Is there a way to obtain a list of all valid passes so that we can show in the help option, or is it too many?

Reply:
Added displaying of available passes in case if supplied pass is unknown to system